perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -201,6 +201,7 @@ sub report_form_ajax : Path('ajax') : Args(0) {
     my $extra_titles_list = $c->cobrand->title_list($c->stash->{all_areas});
 
     my @list_of_names = map { $_->name } values %{$c->stash->{bodies}};
+    my %display_names = map { $_->name => $_->get_cobrand_handler ? $_->get_cobrand_handler->council_name : $_->name } values %{$c->stash->{bodies}};
```

Comment:
This breaks for any cobrand that does not have `council_name`, ie all international ones.